shell/platform/windows/system_utils_winuwp.cc

```diff
@@ -15,20 +15,47 @@ namespace flutter {
 std::vector<LanguageInfo> GetPreferredLanguageInfo() {
   std::vector<std::wstring> languages = GetPreferredLanguages();
   std::vector<LanguageInfo> language_info;
-  // TODO populate via WinRT
+  for (auto language : languages) {
+    language_info.push_back(ParseLanguageName(language));
+  }
   return language_info;
 }
 
 std::vector<std::wstring> GetPreferredLanguages() {
   std::vector<std::wstring> languages;
-  // TODO populate via WinRT
+  languages.push_back(L"en-US");
```

Comment:
Seems like this still needs a TODO.

Reply:
Todo expanded, issue tracking this here: https://github.com/flutter/flutter/issues/74156